backend/edge-rest/src/main/kotlin/io/featurehub/edge/db/sql/DbFeatureUpdateProcessor.kt

```diff
@@ -9,32 +10,32 @@ import io.featurehub.db.model.query.QDbFeatureValue
 import io.featurehub.edge.rest.FeatureUpdatePublisher
 import io.featurehub.mr.messaging.StreamedFeatureUpdate
 
-class DbFeatureUpdateProcessor: FeatureUpdatePublisher {
+class DbFeatureUpdateProcessor(): FeatureUpdatePublisher {
   @Transactional(readOnly = false)
   override fun publishFeatureChangeRequest(featureUpdate: StreamedFeatureUpdate, namedCache: String) {
-    var upd = QDbFeatureValue().feature.key.eq(featureUpdate.featureKey).environment.id.eq(featureUpdate.environmentId).asUpdate()
-
-    if (featureUpdate.lock != null) {
-      upd = upd.set("locked", featureUpdate.lock)
-    }
-
-    if (featureUpdate.updatingValue == true) {
-      var updateValue: String? = determineDefaultValue(featureUpdate)
-
-      upd = upd.set("defaultValue", updateValue)
-    }
-
-    upd.setRaw("version = version + 1")
-
-    if (upd.update() == 0) {
-      val builder = DbFeatureValue.Builder().locked(featureUpdate.lock ?: false)
-      if (featureUpdate.updatingValue == true) {
-        builder.defaultValue(determineDefaultValue(featureUpdate))
-      }
-      builder.environment(QDbEnvironment().id.eq(featureUpdate.environmentId).findOne())
-      builder.feature(QDbApplicationFeature().select(QDbApplicationFeature.Alias.id).key.eq(featureUpdate.featureKey).parentApplication.id.eq(featureUpdate.applicationId!!).findOne())
-      builder.build().save()
-    }
+//    var upd = QDbFeatureValue().feature.key.eq(featureUpdate.featureKey).environment.id.eq(featureUpdate.environmentId).asUpdate()
```

Comment:
Is this commented out meant to be removed?

Reply:
no, because it was too hard to fix in this MR its commented out to fix in a different PR.